cmd/gandi.go

```diff
@@ -23,6 +23,7 @@ type cli struct {
 	Debug         bool             `kong:"short='d',help='Enable debug logging'"`
 	DryRun        bool             `kong:"help='Enable dry run mode'"`
 	APIKey        string           `kong:"env='GANDI_KEY',help='The Gandi LiveDNS API key (may be stored in the GANDI_KEY environment variable)'"`
+	Token         string           `kong:"env='GANDI_PERSONAL_ACCESS_TOKEN',help='The Gandi Personal Access Token (PAT) (may be stored in the GANDI_TOKEN environment variable)'"`
```

Comment:
Hi, there is an inconsistency in the doc: 

```suggestion
	Token         string           `kong:"env='GANDI_PERSONAL_ACCESS_TOKEN',help='The Gandi Personal Access Token (PAT) (may be stored in the GANDI_PERSONAL_ACCESS_TOKEN environment variable)'"`
```

or maybe just `GANDI_PAT` to keep it short?

Reply:
IIRC, I actually suggested to use `GANDI_PERSONAL_ACCESS_TOKEN` instead of `GANDI_PAT` (mainly because it's more explicit and easier to find in our documentation).

Reply:
Typo fixed